docs/griptape-cloud/knowledge-bases/create-knowledge-base.md

```diff
@@ -0,0 +1,7 @@
+# Knowledge Bases
+
+Use [Knowledge Bases](https://cloud.griptape.ai/knowledge-bases) to prep your data ingested from [Data Sources](../data-sources/create-data-source.md) for retrieval. You can specify multiple Data Sources per Knowledge Base in order to access data ingested from different sources all in one place.
```

Comment:
sometimes our links go into the Cloud console, while other times we link off to other documentation. This could be disoncerting. How do we get consistent, or at least alert a user when they're going to one vs. the other?

Reply:
there's not a clean way to do this in purely markdown (like an external link icon or something). I can word the links to cloud differently like adding "in the Griptape Cloud console" to all of them to differentiate. 

Currently, though. The Framework docs also have this same problem. Example: https://docs.griptape.ai/latest/griptape-framework/drivers/web-search-drivers/#duckduckgo

Reply:
Added more words to the links